DEVELOPERS.md

```diff
@@ -99,7 +100,9 @@ psql -c "create database jobserver"
 ##### Restoring Backups
 
 Copies of production can be restored to a local database using a dump pulled from production.
-They can be copied with:
+If you do not have access to pull backups from production you can set up local data by following [data setup section](#data-setup).
```

Comment:
suggestion: a tiny rewrite

```suggestion
If you do not have access to pull production backups, follow the [data setup section](#data-setup) instead of restoring a backup.
```